### DISCUSSION TITLE
Error starting rhvoice demo

### DISCUSSION THREAD

**Discussion by TurekCom** [RHVoice/RHVoice#671] [Q&A]
When I tried to run the rhvoice demo in step two of the voice creation guide, I got this error.
root@grzegorz:~/tts/HTS-demo_CMU-ARCTIC-SLT# ./configure --with-fest-search-path=$(echo ~)/tts/festival/examples \
  --with-sptk-search-path=$(echo ~)/tts/SPTK-3.11/build/bin \
  --with-hts-engine-search-path=$(echo ~)/tts/hts_engine_api110/bin \
  --with-hts-search-path=$(echo ~)/tts/hts23/bin
checking for gawk... gawk
checking for perl... /usr/bin/perl
checking for wc... /usr/bin/wc
checking for text2utt... :
configure: error: Can't find text2utt of festival
root@grzegorz:~/tts/HTS-demo_CMU-ARCTIC-SLT#
How do I fix it to move on?

**Comment by zstanecic** [comment]
Hi,

We don’t use this step at all.

It is not needed.

 

From: TurekCom ***@***.***> 
Sent: Wednesday, December 7, 2022 2:07 PM
To: RHVoice/RHVoice ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [RHVoice/RHVoice] Error starting rhvoice demo (Discussion #671)

 

When I tried to run the rhvoice demo in step two of the voice creation guide, I got this error.
***@***.***:~/tts/HTS-demo_CMU-ARCTIC-SLT# ./configure --with-fest-search-path=$(echo ~)/tts/festival/examples 
--with-sptk-search-path=$(echo ~)/tts/SPTK-3.11/build/bin 
--with-hts-engine-search-path=$(echo ~)/tts/hts_engine_api110/bin 
--with-hts-search-path=$(echo )/tts/hts23/bin
checking for gawk... gawk
checking for perl... /usr/bin/perl
checking for wc... /usr/bin/wc
checking for text2utt... :
configure: error: Can't find text2utt of festival
***@***.***:/tts/HTS-demo_CMU-ARCTIC-SLT#
How do I fix it to move on?
